internal/pkg/mining/worker.go

```diff
@@ -332,7 +327,7 @@ func (w *DefaultWorker) Mine(ctx context.Context, base block.TipSet, nullBlkCoun
 		post = postOut
 	}
 
-	postInfo := block.NewEPoStInfo(post, postRandomness, block.FromFFICandidates(winners...)...)
+	postInfo := block.NewEPoStInfo(post, block.VRFPi(postRandomness), block.FromFFICandidates(winners...)...)
```

Comment:
this is right, but you want the VRF digest, not the VRFPi (the digest is a hash of the Pi)

Reply:
No you do want the VRFPi here, this is what goes on chain which needs to be the proof for verification reasons.

Reply:
One thing that I was meaning to do if I ended up doing this change is realign our types to correctly capture the difference between VRF proofs and digests.  I think it is worth doing to keep it clear which data is a proof (signature output) vs which is a digest (hash of signature output).